dali/base/dadfs.cpp

```diff
@@ -1256,7 +1256,13 @@ static SecAccessFlags getScopePermissions(const char *scopename,IUserDescriptor
 #endif
             user = queryDistributedFileDirectory().queryDefaultUser();
         }
-        perms = querySessionManager().getPermissionsLDAP(queryDfsXmlBranchName(DXB_Scope),scopename,user,auditflags);
+
+        //Create signature
+        CDateTime now;
+        StringBuffer b64sig;
+        createDaliSignature(scopename, user, now, b64sig);
```

Comment:
You do not check the return of createDaliSignature() here. How does the getPermissionsLDAP() work if b64sig is empty?

Reply:
If certificates are not configured, then the signature will not be computed. Server side, (daldap)  it will call getPermissions, which makes sure there is a signature and that certificates are configured before verification

in getPermissions....

       //Check that the digital signature provided by the caller (signature of
        //caller's "scope;username;timeStamp") matches what we expect it to be
        if (!isEmptyString(reqSignature))
        {
            if (nullptr == pDSM)
                pDSM = queryDigitalSignatureManagerInstanceFromEnv();
            if (pDSM && pDSM->isDigiVerifierConfigured())